docs/references/javascript/organization/membership-request.mdx

```diff
@@ -23,11 +23,69 @@ function getMembershipRequests(params?: GetMembershipRequestParams): Promise<Pag
 | `pageSize?` | `number` | A number that indicates the maximum number of results that should be returned for a specific page. |
 | `status?` | `'pending' \| 'accepted' \| 'revoked'` | The status a request can have. |
 
-### Returns
+### `getMembershipRequests()` returns
```

Comment:
In future we should consider cutting some of these "`x()` returns" sections, as they can be quite redundant, such as in this instance.

Reply:
I've created a linear ticket so we can discuss it in triage:
- https://linear.app/clerk/issue/DOCS-5948/cut-redundant-method-returns-headings-in-ref-docs